guide/index.md

```diff
@@ -6,81 +6,218 @@ sidebar_position: 1
 
 Integrate Fishjam Cloud into your React Native application.
 
+:::info
+This guide assumes you're using Expo Bare Workflow
+:::
+
 ### Install package
 
 ```bash npm2yarn
 npm install @fishjam-cloud/react-native-client
 ```
 
-### Fetch participant token
+### Build native dependencies
+
+```bash
+npx expo prebuild --clean
+```
+
+### Get Room Manager URL
+
+Login to [Fishjam Cloud Dashboard](https://fishjam.io/app) and get your Room Manager URL.
+
+## TLDR:
+
+Checkout [complete example below:](#full-example)
 
-Login to [Fishjam Cloud Dashboard](https://fishjam.io/app) and get your Room Manager url. Next, use it to fetch
-participant token for new room:
+## Step by step instructions:
+
+### 1. Fetch peer token
+
+Use your room manager URL to fetch peer token to get a new room:
 
 ```ts
 const response = await fetch(
-  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/?room=ROOM_NAME&user=USER_NAME`,
+  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/*roomName*/users/*username*`,
 );
 
-const { fishjamUrl, participantToken } = await response.json();
+const { url, peerToken } = await response.json();
 ```
 
-### Join Room and start streaming
+### 2. Join Room and start streaming
 
-To start streaming, you have to prepare your camera and join room:
+:::danger
 
-```ts
-import { joinRoom, useCamera } from "@fishjam-cloud/react-native-client";
+If you want to use a device camera, you must first request permission. Keep in mind that this won't work on iOS Simulator.
+Check [this `guide`](./react-native/installation#step-2-configure-app-permissions) for more information.
 
+:::
```

Comment:
I would split it into two boxes: one about permissions, and one about iOS simulator. These are two different things.